scripts/devtool/main.go

```diff
@@ -91,11 +92,11 @@ func main() {
 	yellow.Println("STARTING DOCKER SERVICES")
 	err = pgCmd.Start()
 	if err != nil {
-		panic("could not start db")
+		panic(fmt.Sprintf("could not start db %s", err.Error()))
 	}
 	group.Go(func() error {
 		err := pgCmd.Wait()
-		red.Println("STOPPING DOCKER SERVICES")
+		red.Printf("STOPPING DOCKER SERVICES %s\n", err.Error())
```

Comment:
What if `err == nil`? And could it look weird during a normal shutdown (assuming that returns context cancelled?)